src/Symfony/Component/Messenger/Bridge/Beanstalkd/Tests/Transport/ConnectionTest.php

```diff
@@ -520,4 +606,5 @@ public function testSendWithRoundedDelay()
 
 interface PheanstalkInterface extends PheanstalkPublisherInterface, PheanstalkSubscriberInterface, PheanstalkManagerInterface
 {
+    public function disconnect(): void;
```

Comment:
Version 5 doesn't include this method in any of its interfaces (probably to avoid a BC break), but the implementation does. Version 7 adds it to the interface.